packages/cli/test/test-utils.js

```diff
@@ -124,8 +126,14 @@ exports.givenLBProject = function(rootDir, options) {
   if (sandBoxFiles.length > 0) {
     for (let theFile of sandBoxFiles) {
       const fullPath = path.join(rootDir, theFile.path, theFile.file);
-      if (!fs.existsSync(fullPath)) {
-        fs.writeFileSync(fullPath, theFile.content);
+      const datasourcespath = path.join(rootDir);
+      try {
+        if (!fs.existsSync(fullPath)) {
+          fsextra.ensureDirSync(path.dirname(fullPath));
+          fs.writeFileSync(fullPath, theFile.content);
+        }
+      } catch (e) {
+        console.error(e);
```

Comment:
When the file cannot be created then the test should fail on that error.

Please remove the `try`/`catch` block and let the test runner handle any error thrown by this helper.

Reply:
fixed